internal/pkg/deploy/workload.go

```diff
@@ -11,3 +11,10 @@ type DeleteWorkloadInput struct {
 	EnvName string // Name of the environment the service is deployed in.
 	AppName string // Name of the application the service belongs to.
 }
+
+// DeleteJobInput holds the fields required to delete a job.
+type DeleteJobInput struct {
+	Name    string // Name of the job that needs to be deleted.
+	EnvName string // Name of the environment the job is deployed in.
+	AppName string // Name of the application the job belongs to.
+}
```

Comment:
why do we not use `DeleteWorkloadInput`?

Reply:
good question! changed.